mlflow/metrics/genai/genai_metric.py

```diff
@@ -486,6 +490,11 @@ def make_genai_metric(
         "grading_context_columns": grading_context_columns,
         "include_input": include_input,
         "parameters": parameters,
+        "grading_system_prompt_template": (
+            grading_system_prompt_template.template_strs
+            if isinstance(grading_system_prompt_template, PromptTemplate)
+            else grading_system_prompt_template
```

Comment:
Should we have a guard to ensure that the user-provided input is in the type set of what is defined in arg types?